vmdk_plugin/utils/fs/device_watcher_windows_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// DeviceWatcher unit tests.
+
+package fs
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewDeviceWatcher(t *testing.T) {
+	watcher := NewDeviceWatcher()
+	assert.NotNil(t, watcher, "NewDeviceWatcher must return a watcher instance")
+}
+
+func TestDeviceWatcherInit(t *testing.T) {
+	watcher := NewDeviceWatcher()
+	watcher.Init()
+	select {
+	case event := <-watcher.Event:
+		assert.NotNil(t, event, "Event chan shouldn't return nil")
```

Comment:
Since no disk is being attached as part of this test, this event will timeout after 1 second (as per powershell script timeout.) Please add a comment about it.
This code path will be tested through e2e test.